javascript/javascript2/week3/Fastpresser/index.js

```diff
@@ -0,0 +1,127 @@
+const button = document.getElementById("start-button");
+const p = document.getElementById("count-down");
+const left = document.getElementById("left");
+const right = document.getElementById("right");
+const input = document.getElementById("input");
+const leftPlayer = document.querySelector(".leftPlayer");
+const rightPlayer = document.querySelector(".rightPlayer");
+
+var confettiElement = document.getElementById("my-canvas");
+var confettiSettings = { target: confettiElement };
+var confetti = new ConfettiGenerator(confettiSettings);
+
+function inputValue() {
+  return input.value;
+}
+// Setting a count down until the game will start
+button.addEventListener("click", loopArray);
+
+const array = [
+  function three() {
+    setTimeout(() => {
+      p.innerHTML = 3;
+    }, 1000);
+  },
+  function two() {
+    setTimeout(() => {
+      p.innerHTML = 2;
+    }, 2000);
+  },
+  function one() {
+    setTimeout(() => {
+      p.innerHTML = 1;
+    }, 3000);
+  },
+  function go() {
+    setTimeout(() => {
+      p.innerHTML = "Play";
+    }, 3500);
+  },
+  function enableGame() {
+    button.removeEventListener("click", loopArray);
+  },
+];
+
+function loopArray() {
+  for (i = 0; i < array.length; i++) {
+    array[i]();
+  }
+}
+
+//Counting the number of clicks for each player
+let leftResult = 0;
+let rightResult = 0;
+
+function registerNumberOfClicks() {
+  window.addEventListener("keypress", keyCounter);
+  function keyCounter(event) {
+    if (event.key === "s") {
+      leftResult++;
+      left.innerHTML = leftResult;
+    }
+    if (event.key === "l") {
+      rightResult++;
+      right.innerHTML = rightResult;
+    }
+  }
+  //Timer for how long the game will be and getting the results back
+  const setGameDuration = inputValue() * 1000 + 4000;
```

Comment:
I think 4000ms is not needed here, because the game started after 3, 2, 1, countdown.
So it is just adds 4 extra seconds to the actual game time, which user does not expect :)

Reply:
I first set it as a predifined time for the game, but I can see it was creating a problem.